### DISCUSSION TITLE
Enabling exporting of logs by default being a breaking change?

### DISCUSSION THREAD

**Discussion by jmichalek132** [open-telemetry/opentelemetry-java-instrumentation#8599] [Q&A]
Hi, reading the [release notes](https://github.com/open-telemetry/opentelemetry-java/releases/tag/v1.26.0) for [opentelemetry-java](https://github.com/open-telemetry/opentelemetry-java) do I understand correctly that in 1.27 the otel.logs.exporter  will be switched from none to otlp ? If so this would cause the java agent to start sending logs via OTLP along side metrics and traces. Is that the case? If so this feels like breaking change given most people will already collect logs in a different way.

Moved here from [slack](https://cloud-native.slack.com/archives/C014L2KCTE3/p1685114307719339) as requested.

**Comment by mateuszrzeszutek** [comment]
Thanks for filing this @jmichalek132 !

It'd be great if you could join our [Thursday SIG meeting](https://github.com/open-telemetry/community/#implementation-sigs) to discuss that.

@trask @jack-berg just an idea: since we're already planning to do a 2.0 release in a couple of months (#8280), if this turns out to be a significant problem for most of the users perhaps we could delay that change and include it in the October 2.0 release? (just the javaagent -- we could override the default configuration via the SPI in the agent)

**Reply by jmichalek132** [reply]
I won't be able to join unfortunately, but I will ask one of my colleagues to join instead to explain our case. But yeah delaying the change until 2.0 release seems like a good idea to me.

**Reply by jack-berg** [reply]
I've mentioned this before, but I think we should _definitely_ adjust `otel.logs.exporter` to be `otlp` by default, but we _should_ consider whether we want to install logback / log4j instrumentation by default. Enabling otlp export by default but disabling the instrumentation by default results in no change from the current behavior, but ensures that the log signal is aligned to traces a metrics from a sdk config perspective.
